src/utils/routePaths.ts

```diff
@@ -34,14 +34,23 @@ const routePaths = {
   sprintsPage(teamId: string) {
     return `/my-voyage/${teamId}/sprints`;
   },
+  emptySprintPage(teamId: string, sprintNumber: string) {
+    return `/my-voyage/${teamId}/sprints/${sprintNumber}`;
+  },
+  sprintPage(teamId: string, sprintNumber: string, meetingId: string) {
+    return `/my-voyage/${teamId}/sprints/${sprintNumber}/meeting/${meetingId}`;
+  },
```

Comment:
since there's a routepath name of sprintspage, it might be better to name this something like sprintWeek or sprintNumber, just to make it easier and clearer the difference between the 2